cli/tools/init/templates/deno_lib.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "{{name}}",
+  "version": "1.0.0",
+  "exports": {},
+  "tasks": {
+    "dev": "deno run --watch main.ts"
```

Comment:
**nitpick:** Why does a library default to having a `main.ts`? That sounds more like an application thing.

Reply:
I agree, probably better to do `lib.ts` instead

Reply:
Or maybe even better, `mod.ts`?

Reply:
I've changed this to `mod.ts`, which is the default entrypoint name for libraries in Deno. I also changed the subcommand to `deno test`, rather than `deno run`, as it's a for a library, rather than a script. Happy to change, if there are better suggestions.